lib/logos.js

```diff
@@ -17,6 +17,9 @@ const logoAliases = {
   tfs: 'azure-devops',
 }
 
+// logos which we no longer allow that are available via simple-icons
+const logoBlacklist = ['travis-ci']
```

Comment:
Would it be simpler to block these at load time (i.e. in `load-simple-icons.js`)?